network-mux/network-mux.cabal

```diff
@@ -48,6 +52,10 @@ library
                        Network.Mux.Bearer.Pipe
                        Network.Mux.Bearer.Queues
                        Network.Mux.Bearer.Socket
+                       Network.Mux.DeltaQTraceStats
+                       Network.Mux.DeltaQTraceStatsSupport
+                       Network.Mux.DeltaQTraceTransformer
+                       Network.Mux.DeltaQTraceTypes
```

Comment:
Can we use the hierarchical namespace and use something like
```
                       Network.Mux.DeltaQ.TraceStats
                       Network.Mux.DeltaQ.TraceStatsSupport
                       Network.Mux.DeltaQ.TraceTransformer
                       Network.Mux.DeltaQ.TraceTypes
```

Reply:
Will do